packages/grafana-ui/src/components/Select/IndicatorsContainer.tsx

```diff
@@ -14,6 +14,9 @@ export const IndicatorsContainer = React.forwardRef<HTMLDivElement, React.PropsW
         styles.suffix,
         css`
           position: relative;
+          & > * {
+            margin-left: 2px;
+          }
```

Comment:
think it would be better to set `gap: <gap>;` on the container since it's already flex, rather than setting `margin-left: 2px;` on every immediate child.

Reply:
also might be better to use a theme spacing value here instead of hardcoding `2px`